### PR TITLE
Babel presets fix

### PR DESCRIPTION
**Pull request by lipis**
Should also fix the #791

### CODE REVIEW

File: main/static/src/script/common/util.coffee [hunk 1]
```diff
@@ -43,7 +43,7 @@ window.init_time = ->
         else
           $(this).text date.fromNow()
         $(this).attr 'title', date.local().format 'dddd, MMMM Do YYYY, HH:mm:ss Z'
-      setTimeout arguments.callee, 1000 * 45
+      setTimeout temp_func, 1000 * 45
```
Comment (lipis) [inline main/static/src/script/common/util.coffee]: other ideas are welcome: https://stackoverflow.com/a/12050920

File: main/templates/bit/script.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # if config.DEVELOPMENT
-  <script src="/p/dev/script/ext.js?{{config.CURRENT_VERSION_ID}}"></script>
-  <script src="/p/dev/script/script.js?{{config.CURRENT_VERSION_ID}}"></script>
+  <script src="/p/dev/script/ext.js?{{random()}}"></script>
+  <script src="/p/dev/script/script.js?{{random()}}"></script>
```
Comment (lipis) [inline main/templates/bit/script.html]: I think app engine changed something so it wouldn't update the local version ID on every refresh..

Comment (mdxs) [reply main/templates/bit/script.html]: hmm, that could have some side effects... I would rather like to have this predictable

Comment (lipis) [reply main/templates/bit/script.html]: for localhost..? does it really matter?

Comment (mdxs) [reply main/templates/bit/script.html]: Cannot (yet) reproduce that the ``{{config.CURRENT_VERSION_ID}}`` approach doesn't work.

Comment (lipis) [reply main/templates/bit/script.html]: try to change only js/css and check if the change is actually happening with autorefresh

Comment (mdxs) [reply main/templates/bit/script.html]: You mean on localhost with the development server?

File: package.json [hunk 3]
```diff
@@ -2,12 +2,12 @@
   "author": "Panayiotis Lipiridis <lipiridis@gmail.com>",
   "devDependencies": {
     "babel-core": "6.26.0",
```
[the remark this answers is not in the document]
Comment (mdxs) [reply package.json]: Trying...

[the remark this answers is not in the document]
Comment (mdxs) [reply package.json]: Received an "Error: Cannot find module 'babel-core'" on ``gulp build``, while using:
````
  "devDependencies": {
    "@babel/core": "7.0.0-beta.40",
    "@babel/preset-env": "7.0.0-beta.40",
```

[the remark this answers is not in the document]
Comment (mdxs) [reply package.json]: Perhaps due to ``eslint`` requirement... ``"babel-code-frame": "6.26.0",``

[the remark this answers is not in the document]
Comment (lipis) [reply package.json]: right.. I also tried it yesterday.. but no luck 

https://github.com/babel/gulp-babel#usage

[the remark this answers is not in the document]
Comment (mdxs) [reply package.json]: Could https://github.com/babel/babel-eslint#setup perhaps show a hint... to not use ``strict``?

[the remark this answers is not in the document]
Comment (mdxs) [reply package.json]: Just keep it ``"babel-core": "6.26.0",``

File: package.json [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
-    "babel-preset-es2015": "6.24.1",
+    "@babel/preset-env": "7.0.0-beta.40",
```
Comment (mdxs) [inline package.json]: Just keep it ``"babel-preset-es2015": "6.24.1",``

File: main/static/src/script/common/util.coffee (hunk 1 above)
Comment (mdxs) [reply main/static/src/script/common/util.coffee]: I'm no Javascript expert, but could the "temp_func" in this case read something like "temp_window_init_time" or "window_init_time_func" to make it more readable?

Comment (lipis) [reply main/static/src/script/common/util.coffee]: That actually doesn't work :/

Comment (lipis) [reply main/static/src/script/common/util.coffee]: should be OK now!

**Pull request merged**